### PR TITLE
Correct user setting to system setting

### PR DESCRIPTION
**Pull request by AllForNothing**
1. Correct user setting to Harbor configuration, make it consistent with Harbor UI
2. Add two new configuration items: `audit_log_forward_endpoint ` and `skip_audit_log_database `
Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

**Review by wy65701436 — APPROVED** [review]
lgtm

**Review by MinerYang — APPROVED** [review]
lgtm

File: docs/install-config/configure-system-settings-cli.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
 ---
-title: Configure Harbor User Settings
+title: Harbor Configuration
 weight: 65
 ---
 
-User settings are configured separately from the [system settings](configure-yml-file.md). You can change user settings in the Harbor interface, through HTTP requests, or using an environment variable. This page describes the available user settings, and how to use the commandline or environment variable to update user settings.
+Harbor Configuration is configured separately from the [Configure the Harbor YML File](configure-yml-file.md) section. You can change the configuration in the Harbor interface, through HTTP requests, or using an environment variable. This page describes the available configuration items, and how to use the commandline or environment variable to update the configuration.
```
Comment (stonezdj) [inline docs/install-config/configure-system-settings-cli.md:6]: Some Harbor configuration is configured separately from the [Configure the Harbor YML File](configure-yml-file.md) section...

Comment (AllForNothing) [reply docs/install-config/configure-system-settings-cli.md]: Updated

**Pull request merged**